cernopendata/templates/cernopendata_pages/front/news.html

```diff
@@ -0,0 +1,45 @@
+<div class="homepage-section news section align-items-center" id="news" data-anchor="news">
+    <div class="container-fluid align-items-center">
+            <div class="news-header">
+                <h3 class="">News & Updates</h3>
+            </div>
+        <div class="row justify-content-space-between">
+            {% for article in featured_articles %}
+            {% set article = article._source %}
+            <div class="col-sm col-md-4">
+                <div class="news-card container-fluid">
+                    <div class="card-body">
+                        <p class="row date-author-small">
+                            {{ article.created or '-' }} by
+                            {{ article.author or 'Open Data Portal team'}}
+                        </p>
+                        <div class="row">
+                            <h4>
+                                <a href="/articles/{{article.control_number}}">{{article.title}}</a>
+                            </h4>
+                        </div>
+                        <div class="tags-box row align-self-end">
+                            <a href="/search?type={{article.type.primary}}" class="badge  badge-success custom-badge-success">
+                                {{article.type.primary }}
+                            </a>
+                            {% if article.type.secondary %}
+                                {% for subtype in article.type.secondary %}
+                                    <a href="/search?subtype={{subtype}}" class="badge  badge-info">{{ subtype }}</a>
+                                {% endfor %}
+                            {% endif %}
+                            {% for tag in article.tags %}
+                                <a href="/search?tags={{tag}}" class="badge badge-secondary">{{ tag }}</a>
+                            {% endfor %}
+                            {% if article.experiment %}
+                            <a href="/search?experiment={{article.experiment}}" class="badge badge-warning custom-badge-warning">
+                                {{ article.experiment }}
```

Comment:
Can also have a "More" button here at the end, leading to the search page with `...&type=News`.